File: myblog-server/src/main/java/com/site/blog/client/MyClient.java
package com.site.blog.client;

import com.dtflys.forest.annotation.Request;
import com.site.blog.model.dto.Result;
import com.site.blog.model.entity.Tag;

import java.util.List;

public interface MyClient {

    @Request("http://localhost:2801/v2/home/tags")
    Result<List<Tag>> helloForest();

}

File: myblog-kt/src/main/kotlin/com/site/blog/client/MyClient.kt
package com.site.blog.client

import com.dtflys.forest.annotation.Request
import com.site.blog.model.dto.Result
import com.site.blog.model.entity.Tag

interface MyClient {
    @Request("http://localhost:2801/v2/home/tags")
    fun helloForest(): Result<List<Tag>>
}